### DISCUSSION TITLE
Actividad Docker - Grupo 6

### DISCUSSION THREAD

**Discussion by GonzaloEBaez** [frlp-utn-ingsoft/catedra#118] [Show and tell]
https://github.com/GonzaloEBaez/vetsoft.git

**Comment by rodriguezemautn** [comment]
Correciones:
- verficar el README no son claras las instrucciones. No confindir entre imagen y contenedor
- no despliega el contenedor en modo detach (-d) Al cerrer la consola se cierra el contenedor
- los endpoint dan error, no se migra la base de datos(verificar orden del CMD en el Dockerfile o utilizar un script.sh en un ENTRYPOINT) 
- no sugiere como acceder a la app

Puntos aprobados:
* construye la imagen
* buen tamaño de imagen
*despliega el contenedor sin modo detach (segundo plano) con errores en los endpoint por no migrar la bbdd

Salida de comandos:
$ docker images -a
vetsoft-app   1.0       2a5471178e9a   39 seconds ago   299MB
$ docker ps -a
1c0eb27ae31e   vetsoft-app:1.0   "python manage.py ru…"   2 minutes ago   Exited (0) About a minute ago             vetsoft-v0
